griptape/tasks/prompt_task.py

```diff
@@ -34,7 +34,7 @@ def prompt_stack(self) -> PromptStack:
 
         if memory:
             # inserting at index 1 to place memory right after system prompt
-            stack.add_conversation_memory(memory, 1)
+            memory.add_to_prompt_stack(stack, 1)
```

Comment:
Identical behavior, but the responsibility is better suited for Conversation Memory rather than Prompt Stack. Prompt Stack is intended to be a relatively lightweight abstraction and should not have use-case specific functionality like this in it.